src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinGuiScreenResourcePacks.java

```diff
@@ -0,0 +1,29 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import java.awt.*;
```

Comment:
You are only using `java.awt.Desktop`. Why are you importing everything from that package?

Reply:
When I clip import class on `Desktop it does that` and also when hitting the keybind to optimize import, it changes the Desktop to import all so I didn't pay attention to it really

Reply:
Huh, strange